frontend/src/components/stats/stats_panel.vue

```diff
@@ -0,0 +1,268 @@
+            flex-direction: column;
+            align-items: center;
+            justify-content: center;
+          "
+          class="mx-auto"
+          height="250px"
+          max-width="100%"
+          outlined
+        >
+          <h3>Job progress:</h3>
+          <div v-if="job_data_fetched" style="width: 50%">
+            <pie-chart
+              :data="job_chart.chartData"
+              :options="job_chart.chartOptions"
+            ></pie-chart>
+          </div>
+        </v-card>
+      </v-col>
+      <v-col cols="12" sm="4">
+        <v-card
+          style="
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            justify-content: center;
+          "
+          class="mx-auto"
+          height="250px"
+          max-width="100%"
+          outlined
+        >
+          <h3>My progress:</h3>
+          <br />
+          <br />
+          <div v-if="job_data_fetched" style="width: 80%">
+            <span style="display: flex; justify-content: space-between">
+              Total tasks assigned:
+              <strong>{{ current_user_performance.total }}</strong>
+            </span>
+            <br />
+            <span style="display: flex; justify-content: space-between">
+              Tasks completed:
+              <strong>{{ current_user_performance.completed }}</strong>
+            </span>
+            <br />
+            <span style="display: flex; justify-content: space-between">
+              Annotations created:
+              <strong>{{ current_user_performance.instances }}</strong>
+            </span>
+            <br />
+          </div>
+        </v-card>
+      </v-col>
+      <v-col cols="12" sm="4">
+        <v-card
+          style="
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            justify-content: center;
+          "
+          class="mx-auto"
+          height="250px"
+          max-width="100%"
+          outlined
+        >
+          <v-menu offset-y>
+            <template v-slot:activator="{ on, attrs }">
+              <div v-if="!update_user_cart">
+                <v-avatar size="30" v-bind="attrs" v-on="on" color="indigo">
+                  <v-icon dark> mdi-account-circle </v-icon>
+                </v-avatar>
+                <span v-on="on" style="cursor: pointer">
+                  {{
+                    items.find((item) => item.id === current_user_stat).title
+                  }}
+                </span>
+              </div>
+              <div v-else>
+                <span v-on="on" style="cursor: pointer">
+                  Getting data for
+                  {{
+                    items.find((item) => item.id === current_user_stat).title
+                  }}
+                  ...
+                </span>
+              </div>
+            </template>
+            <v-list>
+              <v-list-item v-for="(item, index) in items" :key="index">
```

Comment:
For this one I would check the `<v-select>` component from vuetify or our wrapper `diffgram_select.vue` I think it will reduce the amount of code for this section of the component.

Reply:
Of course, thanks for the suggestion